apps/examples/testcase/le_tc/kernel/itc_pthread.c

```diff
@@ -101,7 +101,13 @@ static void *thread_set_get_specific(void *param)
 	int exec_index;
 	g_flag = false;
 
+	ret_chk = pthread_key_create(&g_key, NULL);
```

Comment:
Don't you need to put this into for loop?

Reply:
added both create and delete into loop